Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -1217,6 +1217,7 @@ void ACesium3DTileset::updateTilesetOptionsFromProperties() {
       this->_pTileset->getOptions();
   options.maximumScreenSpaceError =
       static_cast<double>(this->MaximumScreenSpaceError);
+  options.maximumCachedBytes = static_cast<int64_t>(this->MaximumCachedBytes);
```

Comment:
You can remove the cast, because the source and destination variables are both int64s now.

Reply:
Fixed ! 👍